aws-lambda-edge-add-hsts-header/main.tf

```diff
@@ -0,0 +1,16 @@
+data "archive_file" "lambda" {
```

Comment:
**[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
data "archive_file" "lambda" is declared but not used